tsinfer/formats.py

```diff
@@ -693,9 +692,8 @@ class SampleData(DataContainer):
         to use for compressing data. Any codec may be used, but
         problems may occur with very large datasets on certain codecs as
         they cannot compress buffers >2GB. If None, do not use any compression.
-        By default, use the :class:`numcodecs.zstd.Zstd` codec is used
-        when data is written to a file, and no compression when data is
-        stored in memory.
+        Default=``DEFAULT_COMPRESSOR``, currently defined as the
```

Comment:
I don't think DEFAULT compressor is actually defined externally, so just leave it as "By default, use the :class:`numcodecs.zstd.Zstd` codec is used".

Reply:
OK, just changed to simply 
```
Default=:class:`numcodecs.zstd.Zstd`.
```